addons/account_edi/models/account_move.py

```diff
@@ -307,6 +367,62 @@ class AccountMoveLine(models.Model):
     # Export Electronic Document
     ####################################################
 
+    def _prepare_edi_vals_to_export(self):
+        ''' Same as '_prepare_edi_vals_to_export' on the invoice but for a single invoice line.
```

Comment:
It could use some explaining about the price without discount, ...  Why the regular fields are not enough.